### PR TITLE
WINC/Health management for Windows nodes -- WMCO 7.0.0

### PR DESCRIPTION
**Pull request by mburke5678**
https://issues.redhat.com/browse/OSDOCS-2755

Previews:
[New images](https://52788--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/understanding-windows-container-workloads.html#windows-workload-management_understanding-windows-container-workloads) Fig 1 updated with WINC and updated to meet current image guidelines. Fig 2 updated to meet current image guidelines only. 
WICD added to [Windows Services table](https://52788--docspreview.netlify.app/openshift-enterprise/latest/windows_containers/understanding-windows-container-workloads.html#windows-node-services_understanding-windows-container-workloads).

QE review:
- [x] QE has approved this change

### CODE REVIEW

File: modules/windows-node-services.adoc [hunk 1]
```diff
@@ -19,8 +19,8 @@ The following Windows-specific services are installed on each Windows node:
 |Container Network Interface (CNI) plug-ins
 |Exposes link:https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#networking[networking] for Windows nodes.
 
-|Windows Machine Config Bootstrapper (WMCB)
-|Configures the kubelet and CNI plug-ins.
+|Windows Instance Config Daemon (WICD)
+|Creates the Windows worker node. Starts the containerd container runtime and the kubelet on the node. Maintains the state of Windows Services on the node using a config map created by the Windows Machine Config Operator (WMCO).
```
Comment (aravindhp) [inline modules/windows-node-services.adoc:23]: WICD basically manages all the services on the Windows instance and does not create the node per se. So I would suggest something like: `Maintains the state of all services running on the Windows instance, defined by WMCO using a ConfigMap, for the instance to function as a worker node`. Definitely needs some word smithing 😄

File: modules/windows-node-services.adoc [hunk 2]
```diff
@@ -22,6 +22,9 @@ The following Windows-specific services are installed on each Windows node:
 |Windows Instance Config Daemon (WICD)
 |Creates the Windows worker node. Starts the containerd container runtime and the kubelet on the node. Maintains the state of Windows Services on the node using a config map created by the Windows Machine Config Operator (WMCO).
 
+|Windows Machine Config Bootstrapper (WMCB)
```
Comment (aravindhp) [inline modules/windows-node-services.adoc:25]: Is there a reason to add WMCB back?

**Review by aravindhp — COMMENTED** [review]
The first commit includes a mocked up image. Is that required? I suggest dropping that from the commit and just introduce the new image in the final commit.

File: modules/windows-node-services.adoc (hunk 2 above)
Comment (mburke5678) [reply modules/windows-node-services.adoc:25]: @aravindhp I didn't remove the WMCB. Apparently I misunderstood what [you said in Slack](https://coreos.slack.com/archives/CM4ERHBJS/p1668542947100979?thread_ts=1668461867.424479&cid=CM4ERHBJS). And, you left WMCB in [your presentation diagrams](https://docs.google.com/presentation/d/1KKjXcxaqUD_2JBnAjwkEv1po2VFGkMgoqHNcyMYBGjs/edit#slide=id.gbd5f5dd60f_0_0).

Comment (aravindhp) [reply modules/windows-node-services.adoc]: Sorry there are multiple diagrams that need to be updated in that deck. [Here](https://docs.google.com/presentation/d/1KKjXcxaqUD_2JBnAjwkEv1po2VFGkMgoqHNcyMYBGjs/edit#slide=id.g18e22cfe629_0_0) is the one that I updated.

Comment (aravindhp) [reply modules/windows-node-services.adoc]: >Apparently I misunderstood what [you said in Slack](https://coreos.slack.com/archives/CM4ERHBJS/p1668542947100979?thread_ts=1668461867.424479&cid=CM4ERHBJS).

Sorry I should have clarified that the WICD work not being fully complete in 7.0.0 only implies WMCO still does some of the work WICD will do in the future. All WMCB responsibilities have been removed in 7.0.0.

**Review by Wiharris — COMMENTED** [review]
/LGTM

**Pull request merged**